test/y2partitioner/widgets/pages/nfs_mounts_test.rb

```diff
@@ -409,6 +415,81 @@ def includes_error_msg?(term)
           expect(nfs_object.mount_point.mount_options).to eq ["nfsvers=4"]
         end
       end
+
+      context "and neither the server nor the directory are modified" do
+        let(:client_result) do
+          {
+            "device" => "srv:/home/a", "fstopt" => "", "mount" => "/foo", "vfstype" => "nfs"
+          }
+        end
+
+        it "does no re-create the NFS mount" do
+          previous_sid = nfs_object.sid
+
+          nfs_page.handle(event)
+
+          sid = Y2Storage::Filesystems::Nfs.find_by_server_and_path(device_graph, "srv", "/home/a").sid
+
+          expect(previous_sid).to eq(sid)
+        end
+
+        it "edits the NFS mount in the current devicegraph" do
+          expect(nfs_object.mount_point.path).to_not eq("/foo")
+
+          nfs_page.handle(event)
+
+          expect(nfs_object.mount_point.path).to eq("/foo")
```

Comment:
you can use here nicer "expect().to change" - https://relishapp.com/rspec/rspec-expectations/v/2-0/docs/matchers/expect-change